daemon/api.go

```diff
@@ -703,6 +733,10 @@ func sideloadSnap(c *Command, r *http.Request, user *auth.UserState) Response {
 		flags |= snappy.DeveloperMode
 	}
 
+	if len(form.Value["try"]) > 0 && form.Value["try"][0] != "" {
```

Comment:
Related to the prior point, it means we're ignoring the "action" field entirely. Can we please take the decision based on it instead?  We should probably also make this work as usual without the multipart form, but we don't need to support that now.